packages/components/organisms/f-cookie-banner/src/components/CookieBanner.vue

```diff
@@ -0,0 +1,364 @@
+            return /(iPhone|iPad).*Safari/.test(navigator.userAgent);
+        },
+        /**
+         * Resend GTM events
+         */
+        resendEvents () {
+            const dataLayer = window.dataLayer || [];
+            const excludeEvents = ['gtm.js', 'gtm.click', 'gtm.linkClick', 'gtm.triggerGroup', 'trackConsent', 'showConsent'];
+            let i;
+            for (i = 0; i < dataLayer.length; i++) {
+                if (dataLayer[i].event && !excludeEvents.includes(dataLayer[i].event)) {
+                    dataLayer[i].eventResent = dataLayer[i].event;
+                    dataLayer[i]['gtm.uniqueEventId'] = Math.floor(100000 + Math.random() * 900000);
+                    dataLayer.push(dataLayer[i]);
+                }
+                if (dataLayer[i].event && dataLayer[i].event === 'trackConsent' && (dataLayer[i].userData.consent === 'full' || dataLayer[i].userData.consent === 'necessary')) {
+                    dataLayer.push({ eventResent: false });
+                    break;
+                }
+            }
+        }
+    }
+};
+</script>
+
+<style lang="scss" module>
+    .c-cookieBanner {
+        font-family: $font-family-base;
+    }
+
+    .c-cookieBanner-overlay {
+        position: fixed;
+        top: 0;
+        bottom: 0;
+        left: 0;
+        right: 0;
+        background-color: rgba(0, 0, 0, 0.7);
+        z-index: 99999991;
+    }
+
+    .c-cookieBanner--hidden {
+        display: none;
+    }
+
+    .c-cookieBanner-card {
+        position: fixed;
+        bottom: 0;
+        left: 0;
+        right: 0;
+        display: flex;
+        flex-direction: row;
+        justify-content: space-between;
+        background-color: $white;
+        z-index: 99999992;
+    }
+
+    .c-cookieBanner-CTA {
+        min-width: 352px;
+    }
+
+    .c-cookieBanner-text {
+        margin: 0;
+        padding: 0;
+    }
+
+    .c-cookieBanner-link {
+        color: $color-link-default;
+    }
+
+    .c-cookieBanner-title {
+        @include font-size(heading-m);
+        font-weight: $font-weight-bold;
+        margin: spacing() 0;
+        padding: 0;
+        color: $color-headings;
+        &:hover,
+        &:focus {
+            a {
+                text-decoration: none;
+            }
+        }
+    }
+
+    .c-cookieBanner-content,
+    .c-cookieBanner-CTA {
+        margin: 0 auto;
+        padding: spacing(x4);
+    }
+
+    @media (max-width: 768px) {
+        .c-cookieBanner-card {
+            flex-direction: column;
+            padding: spacing(x2) 0;
+        }
+
+        .c-cookieBanner-ios {
+            padding-bottom: 80px;
+        }
+
+        .c-cookieBanner-content,
+        .c-cookieBanner-CTA {
+            padding: spacing(x0.5) spacing(x3);
+        }
+
+        .c-cookieBanner-CTA {
+            min-width: initial;
+            display: flex;
+            flex-direction: row-reverse;
+            margin: 0;
+        }
+    }
+
+    @media (max-width: 600px) {
```

Comment:
Do we have breakpoints somewhere we can reuse? 🤔

Reply:
> https://github.com/justeat/fozzie/blob/master/src/scss/settings/_variables.scss#L76-L83

```suggestion
    @media ('<narrowMid') {
```